app/helpers/external_logout_helper.rb

```diff
@@ -0,0 +1,15 @@
+module ExternalLogoutHelper
+  def external_logout_props
+    {
+      version: SETTINGS[:version].version,
```

Comment:
there is an option to get foreman version without a round trip to the server via `package.json`

```js
import { version } from '../package.json';

console.log(version);
```

Reply:
This is a different version,
it can be anything the user have configured under `SETTINGS`
maybe there is a way to update `package.json` with the user version?

Reply:
fun fact, every API response foreman adds a header with the version - see https://github.com/theforeman/foreman/blob/develop/app/controllers/api/base_controller.rb#L227

Reply:
Or it is already in redux's `store` in metadata [`app`](https://github.com/theforeman/foreman/blob/870c50ad43769b059f363c87d4a2f6c702e5ea9e/webpack/assets/javascripts/react_app/ReactApp/index.js#L26) :stuck_out_tongue_closed_eyes: 
Fetched in `ReactApp` and comming from https://github.com/theforeman/foreman/blob/870c50ad43769b059f363c87d4a2f6c702e5ea9e/app/helpers/application_helper.rb#L532
Funny though: would it be helpful to document such reusable items we have around in redux to our storybook? I stumbled upon this just by accident :thinking:
Maybe set of global selectors could do the job?

Reply:
> Or it is already in redux's store in metadata app.metadata

@ezr-ondrej  Only when the page is first mounted by react-router,
since this isn't the case in this PR.. right now this page is rendered from rails.